src/Microsoft.SqlTools.ServiceLayer/LanguageServices/AutoCompleteHelper.cs

```diff
@@ -680,5 +680,79 @@ private static string GetCompletionItemInsertName(Declaration autoCompleteItem)
                 return null;
             }
         }
+
+        /// <summary>
+        /// Converts a SQL Parser List of MethodHelpText objects into a VS Code SignatureHelp object
+        /// </summary>
+        internal static SignatureHelp ConvertMethodHelpTextListToSignatureHelp(List<Babel.MethodHelpText> methods, Babel.MethodNameAndParamLocations locations, int line, int column)
```

Comment:
>methods [](start = 106, length = 7)

Please do argument validation on these (e.g. Validate.IsNotNull()). Internal is more or less equivalent to public, in that we shouldn't trust the caller does the right thing.